### ISSUE TITLE
Iteration 20200804

### ISSUE THREAD

**Issue by jmborr** [neutrons/PyRS#648]
### Review
43 PR's closed
- backend:
  + refactoring and testing
  + bug fixes
  + improved API documentation
- frontend:
  + specific improvements
  + bug fixes
  + additional testing

### Retrospective
- good amount of work
- communication level among team members is good
- Detailed user stories for saving summary and full info to CSV unveiled requirements not taken into account by the design.
  + refactoring of current design could've possibly been avoided if the detailed user stories were made available earlier.
  + On the other hand, a need for these detailed user stories was made evident only after we had a working version of the stress calculation and we realized its limitations.

#### Two to three days
- go over Pete's changes to `StrainField` and `StressField`
- identify in these changes what could be merged into master, thus making the new functionality available to the front end and the CSV exporter
#### One week
- merge new `StrainField` and `StressField` that can access individual `PeakCollection` info
- Integration tests
- Full CSV exporter made now possible
- Load/Save to CVS made possible in the GUI
### Two week
- merge recalculation of `StressField` after `d_0` modification
- visualization of the new stress made possible now in the GUI

Optimization is taken a back seat, since the user rarely modifies `d_0`. It is much more important to have a running implementation.
